README.md

```diff
@@ -4,17 +4,13 @@ Infrastructure for the self-hosted, single-sign-on, community-run services.
 
 * Set the domain name in `env.production`
 * Create the DNS entries in the domain for `login`, `cloud`, `matrix`, `dashboard`, `docs` and maybe more.
-* Install dependencies:
+* Install dependencies (note that `docker-compose 1.25` breaks environment variables as we use them):
 
 ```
-apt install jq docker-compose
-apt install prometheus
+apt install python3-pip prometheus
+pip3 install docker-compose
```

Comment:
suggest using `python3 -m pip install`, as per the pip documentation https://pip.pypa.io/en/stable/installation/#upgrading-pip and this article https://snarky.ca/why-you-should-use-python-m-pip/